cli_test2.py

```diff
@@ -0,0 +1,49 @@
+'''
```

Comment:
I actually prefer ''' since it helps me distinguish the syntax from C#/Java in my head in a way - When I use the " char, my head is automatically tuned into thinking "Oh that's gonna be a string variable". This doesn't happen to me as much with the ' char because it's less often used. ... It's hard to explain, but it's ultimately just my personally preference. If you strongly prefer using """ though, I'm fine with that too.

And well, there's also the lazy reason of not having to hold down the [shift] key when typing the ' char.

Reply:
Okay, sure. We can keep using '''